src/main/java/lotto/money/Money.java

```diff
@@ -0,0 +1,56 @@
+package lotto.money;
+
+import lotto.ticket.TicketValidation;
+
+import java.util.Objects;
+
+public class Money {
+    public static final String ERROR_MESSAGE_MINIMUM_MONEY = "1000원 이상의 금액이 필요합니다.";
```

Comment:
```suggestion
    public static final String ERROR_MESSAGE_MINIMUM_MONEY = MINIMUM_PRICE + "원 이상의 금액이 필요합니다.";
```

상수를 사용한다면 금액이 변경됐을 때 메시지를 변경하지 않아도 되겠네요!